designer/client/src/stylesheets/variables.ts

```diff
@@ -0,0 +1,4 @@
+export const variables = {
```

Comment:
Based on the name of this component, it sounds like an MUI list divider, we can use it here. https://mui.com/material-ui/react-divider/